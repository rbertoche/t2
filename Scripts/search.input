#Cria usuarios
register a
register b
register c
login c
#Faz busca com usarios vazios
search
#Edita um usario
editme
Carlos
Sex
20
#Faz busca outra vez
search
#Desloga, edita outro usuario
logout
login a
editme
Andre
None
27
#Faz busca por idade, retornando a e c
search -a 20:27
#A mesma busca, mas filtrando tambem interesse
search -a 20:27 -i None
#Cria e edita varios usuarios
logout
register 1
login 1
editme
1
None
1
logout
register 3
login 3
editme
3
Sex
3
logout
register 5
login 5
editme
5
Work
5
logout
register 6
login 6
editme
6
Friends
6
logout
#Printa todos eles
login a
search
#Printa em uma certa idade
search -a 3:9
#Printa nenhum
search -a 3:9 -i None
#Testa exibicao de amigos
addfriend 3 5
search -f -a 3:9

