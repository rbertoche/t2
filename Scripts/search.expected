Novo usuario criado.
Novo usuario criado.
Novo usuario criado.
a

Idade: 0
Interesse: Invalid

b

Idade: 0
Interesse: Invalid

c

Idade: 0
Interesse: Invalid

Entre um novo nome ate 50 caracteres ou pressione enter
Entre um novo interesse ou pressione enter
Entre uma nova idade ou pressione enter
Perfil salvo.
a

Idade: 0
Interesse: Invalid

b

Idade: 0
Interesse: Invalid

c
Carlos
Idade: 20
Interesse: Sex

bye c
Entre um novo nome ate 50 caracteres ou pressione enter
Entre um novo interesse ou pressione enter
Entre uma nova idade ou pressione enter
Perfil salvo.
a
Andre
Idade: 27
Interesse: None

c
Carlos
Idade: 20
Interesse: Sex

a
Andre
Idade: 27
Interesse: None

bye a
Novo usuario criado.
Entre um novo nome ate 50 caracteres ou pressione enter
Entre um novo interesse ou pressione enter
Entre uma nova idade ou pressione enter
Perfil salvo.
bye 1
Novo usuario criado.
Entre um novo nome ate 50 caracteres ou pressione enter
Entre um novo interesse ou pressione enter
Entre uma nova idade ou pressione enter
Perfil salvo.
bye 3
Novo usuario criado.
Entre um novo nome ate 50 caracteres ou pressione enter
Entre um novo interesse ou pressione enter
Entre uma nova idade ou pressione enter
Perfil salvo.
bye 5
Novo usuario criado.
Entre um novo nome ate 50 caracteres ou pressione enter
Entre um novo interesse ou pressione enter
Entre uma nova idade ou pressione enter
Perfil salvo.
bye 6
a
Andre
Idade: 27
Interesse: None

b

Idade: 0
Interesse: Invalid

c
Carlos
Idade: 20
Interesse: Sex

1
1
Idade: 1
Interesse: None

3
3
Idade: 3
Interesse: Sex

5
5
Idade: 5
Interesse: Work

6
6
Idade: 6
Interesse: Friends

3
3
Idade: 3
Interesse: Sex

5
5
Idade: 5
Interesse: Work

6
6
Idade: 6
Interesse: Friends

Nenhum usuario casou com a busca.
Amigo adicionado com sucesso.
Amigo adicionado com sucesso.
3
3
Idade: 3
Interesse: Sex

5
5
Idade: 5
Interesse: Work


